webui/quasar.conf.js

```diff
@@ -114,6 +120,10 @@ module.exports = function (ctx) {
     supportIE: false,
 
     build: {
```

Comment:
Hello @andsarr, thank you for your job on this.
I just have a small suggestion to add `compilerOptions.isCustomElement` to get rid of this warning:

![image](https://github.com/traefik/traefik/assets/70909035/bb369f19-7149-4b6f-b44e-8505679c5674)

```suggestion
    build: {
     viteVuePluginOptions: {
        template: {
          compilerOptions: {
            isCustomElement: (tag) => tag.startsWith('hub-')
          }
        }
      },
```